### PR TITLE
Renamed playback/speed to playback/speed_scale

### LINKED ISSUE #2060: Tween speed
I'm using Tween quite offen in my work. But to be honest, I'm not quite sure what does the "set_speed" really stands for - which is crusial. Ok - it makes it faster or slower but I cant find what is the unit here. Wouldn't it be better to change the approach here and swap it with something like "set_tween_time/lengh" or "set_duration" in seconds/miliseconds ? It would be much more intuitive and self documentating.

### PR DESCRIPTION
**Pull request by need12648430**
Also corrected a typo in the respective documentation

Closes #2060

### CODE REVIEW

**Comment by vnen** [comment]
Note that this will break compatibility.

**Comment by need12648430** [comment]
@vnen Should it be tagged for 3.0 milestone, then?

File: scene/animation/tween.cpp [hunk 1]
```diff
@@ -108,8 +108,8 @@ bool Tween::_set(const StringName& p_name, const Variant& p_value) {
 
 	String name=p_name;
 
-	if (name=="playback/speed" || name=="speed") { //bw compatibility
-		set_speed(p_value);
+	if (name=="playback/speed_scale" || name=="speed_scale") { //bw compatibility
```
Comment (akien-mga) [inline scene/animation/tween.cpp]: The comment says "backwards compatibility", so it's not meant to be modified to the new name, it should actually be extended to support for newly deprecated name.

Comment (need12648430) [inline scene/animation/tween.cpp]: Oh, fair. Sorry. :p

**Comment by TheoXD** [comment]
What about consistency, AnimationPlayer also has set_speed() as I don't want to remember which one has "_scale" in the end and which doesn't.

**Comment by bojidar-bg** [comment]
What is the status on this one?

**Comment by akien-mga** [comment]
The way properties are addressed changes for Godot 3.0, and this one will probably become `playback_speed_scale` or just `speed_scale` instead of `playback/speed`. I've mentioned it on #5696 so that we remember to fix it - in the meantime this PR can IMO be closed as we'll do all the renaming work in one batch.

**Pull request closed without merging**